DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/shannonization/src/shannonization.cpp

```diff
@@ -268,28 +268,28 @@ int main(int argc, char** argv) {
 
     bool success = true;
 
-  // Instantiate multiple versions of the kernel
-  // The II achieved by the compiler can differ between FPGA architectures
-  //
-  // On Arria 10, we are able to achieve an II of 1 for versions 1 and 2 of
-  // the kernel (not version 0).
-  // Version 2 of the kernel can achieve the highest Fmax with 
-  // an II of 1 (and therefore has the highest throughput).
-  // Since this tutorial compiles to a single FPGA image, this is not
-  // reflected in the final design (that is, version 1 bottlenecks the Fmax
-  // of the entire design, which contains versions 0, 1 and 2).
-  // However, the difference between versions 1 and 2
-  // can be seen in the "Block Scheduled Fmax" columns in the 
-  // "Loop Analysis" tab of the HTML reports.
-  //
-  // On Stratix 10 and Agilex, the same discussion applies, but version 0
-  // can only achieve an II of 3 while versions 1 and 2 can only achieve
-  // an II of 2. On Stratix 10 and Agilex, we can achieve an II of 1 if we use
-  // non-blocking pipe reads in the IntersectionKernel, which is shown in
-  // version 3 of the kernel.
-  //
+    // Instantiate multiple versions of the kernel
+    // The II achieved by the compiler can differ between FPGA architectures
+    //
+    // On Arria 10, we are able to achieve an II of 1 for versions 1 and 2 of
+    // the kernel (not version 0).
```

Comment:
Is this comment still accurate?  If I undersand the code change you made below it seems like you're now asking for an II of 1 on version 0 of the kernel?

Reply:
Fixed!